version.php

```diff
@@ -4,7 +4,10 @@
 
 /** @var object $plugin */
 $plugin->component = 'assignsubmission_pxaiwriter';
-$plugin->version = 2024120900;
+$plugin->version = 2025121000;
 $plugin->requires = 2021051700; // Moodle 3.11
-$plugin->release = '1.7.0 (Build: 2024-12-09)';
+$plugin->release = '1.7.1 (Build: 2025-12-10)';
 $plugin->maturity = MATURITY_STABLE;
+$plugin->dependencies = [
+    'local_mxaimanager' => 2025111900,
+];
```

Comment:
pxaiwriter requires Moodle 3.11 but local_mxaimanager minimum requires 44 (or 50, depending on branch used) - I think best practice would be to create a new branch of pxaiwriter too